kibana-notebooks/server/adaptors/default_backend.ts

```diff
@@ -275,6 +276,9 @@ export class DefaultBackend implements NotebookAdaptor {
       if (inputType === 'VISUALIZATION') {
         paragraphType = 'VISUALIZATION';
       }
+      if (paragraphInput.substring(0, 3) === '%sql' || paragraphInput.substring(0, 3) === '%ppl') {
+        paragraphType = 'QUERY';
+      } 
```

Comment:
I wonder if we need to make the keywords that indicates the language case sensitive, like should '%SQL' work or not?

Reply:
Good question- will have to follow up with UX on this, all mock-ups and placeholder text use exclusively lowercase